build.gradle

```diff
@@ -10,6 +10,8 @@ plugins {
     id 'application'
     id 'com.diffplug.gradle.spotless' version '3.27.2'
     id 'com.github.spotbugs' version '4.2.0'
+    // Fix guava -jre vs -android dependency resolution.
```

Comment:
Is this just for CRL? If so, could you note that in the comment.

Reply:
Left a better note. Warning, it's a rabbit hole.